applications/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -178,21 +278,20 @@ static void select_next(void)
 	struct ble_peer_operation_event *event = new_ble_peer_operation_event();
 
 	event->op = PEER_OPERATION_SELECT;
-	event->arg = tmp_peer_id;
+	event->bt_app_id = tmp_peer_id;
+	event->bt_stack_id = get_bt_stack_peer_id(tmp_peer_id);
 
 	EVENT_SUBMIT(event);
 }
 
 static void store_peer_id(u8_t peer_id)
 {
 	if (IS_ENABLED(CONFIG_SETTINGS)) {
-		char key[MAX_KEY_LEN];
-		int err = snprintk(key, sizeof(key), MODULE_NAME "/%s",
-				   PEER_ID_STORAGE_NAME);
+		static_assert(MAX_KEY_LEN >
+			      sizeof(MODULE_NAME "/" PEER_ID_STORAGE_NAME), "");
+		char key[MAX_KEY_LEN] = MODULE_NAME "/" PEER_ID_STORAGE_NAME;
```

Comment:
maybe `key[]` since array is initialized with known, constant strings?

Reply:
ok, changed